runtime.go

```diff
@@ -1400,8 +1402,8 @@ func New() *Runtime {
 
 // NewWithContext creates an instance of a Javascript runtime that can be used to run code. Multiple instances may be created and
 // used simultaneously, however it is not possible to pass JS values across runtimes.
-func NewWithContext(ctx context.Context) *Runtime {
-	r := &Runtime{ctx: ctx}
+func NewWithContext(ctx context.Context, shouldForceMemCheck bool) *Runtime {
```

Comment:
if we choose to add a bool here for forcing mem check, we should update the comment.